src/core/tracking.js

```diff
@@ -159,9 +171,34 @@ export function getAction(type) {
   return (
```

Comment:
I find the name of this function confusing, because it doesn't return an "action" at all, it returns a code for a type of add-on (e.g., `addon`, `addon:statictheme`, etc.). So maybe it should be `getAddonTypeForAction` or something like that?

Also, we should have unit tests for this function.

Reply:
I think it was getting the action for the event but I see what you're saying. I can def update as you suggest..

Reply:
Hmm, I know I suggested `getAddonTypeForAction`, but looking at the code again the word "action" is really just being used because we're using this value to populate the `action` argument of the `tracking.sendEvent` function. But it doesn't really represent at action at all. What is represents is the type of add-on being used to record a tracking event. So maybe `getAddonTypeForTracking` makes more sense. What do you think?